packages/mask/src/social-network-adaptor/twitter.com/injection/Avatar/index.tsx

```diff
@@ -47,7 +46,6 @@ export async function injectAvatar(signal: AbortSignal) {
                     ? Plugin.SNSAdaptor.AvatarRealmSourceType.Suggestion
                     : Plugin.SNSAdaptor.AvatarRealmSourceType.Post
 
-                const isVerified = isVerifiedUser(ele)
```

Comment:
The Twitter API requests will increase rapidly w/o this patch. Please keep it in another place.